src/main/java/org/testcontainers/containers/TarantoolSimpleContainer.java

```diff
@@ -0,0 +1,104 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.BuildImageResultCallback;
+import com.github.dockerjava.api.model.Image;
+import com.github.dockerjava.core.DockerClientBuilder;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+
+import static org.testcontainers.containers.PathUtils.normalizePath;
+
+public class TarantoolSimpleContainer extends GenericContainer<TarantoolSimpleContainer> {
```

Comment:
Maybe we just extend TarantoolContainer and add caching + SDK support ? 
The name of class must be changed